pkg/controller/deployment/unsupported_config_overrides_test.go

```diff
@@ -32,29 +37,31 @@ func TestUnsupportedConfigOverrides(t *testing.T) {
 		"Other config overrides should not touch the controller": {
 			overrides: &v1alpha1.UnsupportedConfigOverrides{
 				Webhook: v1alpha1.UnsupportedConfigOverridesForCertManagerWebhook{
-					Args: []string{"test"},
+					Args: testArgs,
 				},
 				CAInjector: v1alpha1.UnsupportedConfigOverridesForCertManagerCAInjector{
-					Args: []string{"test"},
+					Args: testArgs,
 				},
 			},
 			wantArgs: defaultArgs,
 		},
 		"Controller overrides should work": {
 			overrides: &v1alpha1.UnsupportedConfigOverrides{
 				Controller: v1alpha1.UnsupportedConfigOverridesForCertManagerController{
-					Args: []string{"test"},
+					Args: testArgs,
 				},
 			},
-			wantArgs: []string{"test"},
+			wantArgs: defaultPlusTestArgs,
```

Comment:
This only tests `append`  right?  because `testArgs` isn't a valid value in `deploymentFile`.

Reply:
Yes, that was correct. I added more test cases that covers the append as well as replace related behavior of UnsupportedConfigOverrides.